tools/tgs4_scripts/PreCompile.sh

```diff
@@ -73,7 +73,7 @@ apt-get install -y cmake build-essential gcc-multilib g++-multilib cmake wget
 # update auxmos
 if [ ! -d "auxmos" ]; then
 	echo "Cloning Auxmos..."
-	git clone https://github.com/BeeStation/auxmos
+	git clone https://github.com/Putnam3145/auxmos
```

Comment:
We are not going to be cloning off an unknown and untested repo for this.
They could inject files at any time without our control.

If we are going to be testing this, it will need to be a monkestation owned fork of this code.

Reply:
fair i just didn't have any to build off of at the moment and don't have the control to create a monkestation fork of it but yea beestation is forked from putnam's repo so i used it as a placeholder